docs/tutorials/best-practices/speed-up-templates.md

```diff
@@ -0,0 +1,138 @@
+# Speed up your Coder templates and workspaces
+
+October 31, 2024
+
+---
+
+If it takes your workspace a long time to start, find out why and make some
+changes to your Coder templates to help speed things up.
+
+## Monitoring
+
+You can monitor [Coder logs](../../admin/monitoring/logs.md) through the
+system-native tools on your deployment platform, or stream logs to tools like
+Splunk, Datadog, Grafana Loki, and others.
+
+### Coder Observability Chart
+
+Use the [Observability Helm chart](https://github.com/coder/observability) for a
+pre-built set of monitoring integrations.
+
+With the Observability Helm chart, you can monitor [which specific startup
+metrics are a part of the Helm chart?]
+
+To install it with Helm:
+
+```shell
+helm repo add coder-observability https://helm.coder.com/observability
+helm upgrade --install coder-observability coder-observability/coder-observability --version 0.1.1 --namespace coder-observability --create-namespace
+```
+
+### Enable Prometheus metrics for Coder
+
+Our observability bundle gives you this for free which is nice and has
+instructions on how to do it with Coder
+
+[Prometheus.io](https://prometheus.io/docs/introduction/overview/#what-is-prometheus)
+is included as part of the [observability chart](#coder-observability-chart).It
+offers a variety of
+[available metrics](../../admin/integrations/prometheus.md#available-metrics).
+
+You can
+[install it separately](https://prometheus.io/docs/prometheus/latest/getting_started/)
+if you prefer.
+
+### Workspace build timeline
+
+Use the **Build timeline** to monitor the time it takes to start specific
+workspaces. Identify long scripts, resources, and other things you can
+potentially optimize within the template.
+
+![Screenshot of a workspace and its build timeline](../../images/best-practice/build-timeline.png)
+
+Adjust this request to match your Coder access URL and workspace:
+
+```shell
+curl -X GET http://coder-server:8080/api/v2/workspacebuilds/{workspacebuild}/timings \
```

Comment:
```suggestion
curl -X GET https://coder.example.com/api/v2/workspacebuilds/{workspacebuild}/timings \
```
I think we should stay consistent and use `coder.example.com` as the $CODER_URL placeholder throughout docs.

Reply:
++